packages/@okta/i18n/src/properties/login_da.properties

```diff
@@ -221,7 +221,7 @@ factor.oktaVerifyPush = Okta Verify
 factor.push.description = Brug en pushmeddelelse sendt til den mobile app
 factor.duo = Duo Security
 factor.duo.description = Brug pushmeddelelse, SMS eller taleopkald til godkendelse.
-factor.sms = SMS Authentication
+factor.sms = SMS godkendelse
```

Comment:
@anipendakur-okta This is the fix for https://oktainc.atlassian.net/browse/OKTA-402542.